### ISSUE TITLE
Search by alternate titles

### ISSUE THREAD

**Issue by djoole** [Radarr/Radarr#3346]
Hi!
It would be great if radarr would search indexers by alternate titles (when nothing is found with primary title, or always, to be discussed...).

**Comment by djoole** [comment]
It's said [here](https://github.com/Radarr/Radarr/issues/334#issuecomment-334948264) that radarr search 5 alternative titles... seems not to be the case anymore with version 0.2.0.1217.

For instance for _**Dolphins (2018)**_, it only looks for _Dolphins_, and not for _Blue_ (one of the alternative titles on TMDB)

**Comment by djoole** [comment]
@galli-leo, thanks for your quick feedback, but i don't think imdb id search is the problem.
Even if I don't activate "Search by Title", radarr seems to search by title.
But the problem is not that it doesn't search by imdb id, it's that it doesn't search with other title than main title.
At least nothing of that sort is visible in the logs : 
```
19-2-2 17:46:23.7|Trace|Http|Req: 3066 [GET] /api/release?movieId=4&sort_by=releaseWeight&order=asc
19-2-2 17:46:24.0|Trace|Owin|SQLite warning (284): automatic index on MovieFiles(MovieId)
19-2-2 17:46:24.0|Info|NzbSearchService|Searching 1 indexers for [Dolphins]
19-2-2 17:46:24.0|Debug|Torznab|Downloading Feed http://192.168.1.2:9117/api/v2.0/indexers/yggtorrent/results/torznab/api?t=search&cat=102183,102181,5000&extended=1&apikey=(removed)&offset=0&limit=100&q=dolphins%202018
19-2-2 17:46:24.1|Trace|HttpClient|Req: [GET] http://192.168.1.2:9117/api/v2.0/indexers/yggtorrent/results/torznab/api?t=search&cat=102183,102181,5000&extended=1&apikey=(removed)&offset=0&limit=100&q=dolphins%202018
19-2-2 17:46:24.1|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
19-2-2 17:46:27.2|Trace|HttpClient|Res: [GET] http://192.168.1.2:9117/api/v2.0/indexers/yggtorrent/results/torznab/api?t=search&cat=102183,102181,5000&extended=1&apikey=(removed)&offset=0&limit=100&q=dolphins%202018: 200.OK (3171 ms)
19-2-2 17:46:27.3|Debug|NzbSearchService|Total of 0 reports were found for [Dolphins] from 1 indexers
19-2-2 17:46:27.3|Info|DownloadDecisionMaker|No results found
19-2-2 17:46:27.3|Trace|Http|Res: 3066 [GET] /api/release?movieId=4&sort_by=releaseWeight&order=asc: 200.OK (3514 ms)
19-2-2 17:46:27.3|Debug|Api|[GET] /api/release?movieId=4&sort_by=releaseWeight&order=asc: 200.OK (3514 ms)
```
In the log above I only see radarr searchin "dolphins 2008", not "blue 2018"

**Comment by galli-leo** [comment]
@djoole Is that title registered inside Radarr? I. E. Can you see it in the alternative titles section?

**Comment by djoole** [comment]
@galli-leo I wasn't aware of the "title" section in radarr. No the title "Blue" is not registered. No title at all is registered in this section. Why is that?

**Comment by galli-leo** [comment]
@djoole It get's populated by TMDB so that's why Radarr isn't searching the alternative title, it doesn't know about that title.

**Comment by djoole** [comment]
But "Blue" is referenced in TMDB as alternative title (it's the first thing i've checked and mentioned in my first comment) : 

https://www.themoviedb.org/movie/454291-dolphins

**Comment by djoole** [comment]
Another title problem (let me know if you want me to open another issue for this one).
I've searched in radarr the french film "Mia et le lion blanc", and found it. Added it, and it was added as the alternative US title "Mia and the white lion". Nothing in "titles" section.
And radarr pass the title "mia white lion" to jackett instead of the original title "Mia et le lion blanc".

So my understanding is : 
When a movie is originally from an english speaking country, radarr uses the original title (for displaying in UI and passing to indexer)
When a movie is originally from a non speaking country, radarr uses the alternative english title (for displaying in UI and passing to indexer)
Am I correct?

I think it would be more logic to use the UI language for display, and always the original title for passing to indexer, amongst some alternative titles.

And for special cases, it would be great to have the possibility to add a custom alternative title for search.

**Comment by djoole** [comment]
@galli-leo regarding the first problem (alternative title "Blue" not showing for the movie "Dolphins"), I've had a look at the sqlite table "AlternativeTitles", and it only shows a few alternative titles for some movies, and indeed these titles are seen in the "titles" section of each movie.

Now the question is, why does radarr register alternative titles for some movies and non for others...

**Comment by galli-leo** [comment]
Yes then the problem is, that the other title is french. Radarr uses the language of your profile to determine which alternative titles to use for searching. What you have suggested is already tracked on other issues and will make its way into Radarr some day.

**Comment by djoole** [comment]
@galli-leo Would a workaround be to set French as language in my profile? If so, how can I do that? I can't find language settings in radarr.

Can you please point to another issue where the problem is tracked? I would like to subscribe to it.

**Comment by djoole** [comment]
Aaawwwwww, the profile I use was set to language "any"! I've set it to language "french", and now it behaves perfectly :)
Radar first look for original title (english), then french title.

@galli-leo why didn't you said so earlier :P
